.meta/dietpi-imager

```diff
@@ -261,6 +261,35 @@ Do you want to overwrite or backup the existing file to /root/$OUTPUT_IMG_NAME.$
 		IMAGE_SIZE=$(( ( $(sfdisk -qlo End $FP_SOURCE | tail -1) + 1 ) * 512 )) # 512 byte sectors => Byte
 		IMAGE_SIZE=$(( $IMAGE_SIZE + ( 512 * 256 ) )) # 64 byte for secondary GPT + safety net
 
+		# Check if there is enough free disk space to store the image and the 7z file
+		# Image file source
+		if [[ $FP_SOURCE_IMG ]]; then
+
+			NEEDED_FREE_SPACE=$(( $IMAGE_SIZE / 100 * 15 ))
```

Comment:
@sal666 
You assume 7z compression ratio 15:100 right? Sounds realistic, however what you think about addition +100 MiB fixed (in every case) so we avoid to fill the partition to its limit? Could be done, with below suggestion:
```
G_CHECK_FREESPACE /root $(( $NEEDED_FREE_SPACE + 100 )) || Exit_On_Fail # Assure +100 MiB left partition space
```

Reply:
Yep, sure